worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeBiomeRegistry.java

```diff
@@ -72,12 +72,13 @@ static void populate() {
         Map<Integer, BiomeGenBase> biomes = HashBiMap.create();
         Map<Integer, BiomeData> biomeData = new HashMap<Integer, BiomeData>();
 
-        for (BiomeGenBase biome : BiomeGenBase.getBiomeGenArray()) {
+        for (BiomeGenBase biome : BiomeGenBase.biomeRegistry) {
             if ((biome == null) || (biomes.containsValue(biome))) {
                 continue;
             }
-            biomes.put(biome.biomeID, biome);
-            biomeData.put(biome.biomeID, new ForgeBiomeData(biome));
+            int biomeId = BiomeGenBase.getIdForBiome(biome);
```

Comment:
I'm talking about the `biomes` and `biomeData` maps. They keep fewer ForgeBiomeDatas from being created since `populate` stores them.